configure/ingress-nginx/mandatory.yaml

```diff
@@ -206,12 +208,16 @@ spec:
         app.kubernetes.io/part-of: ingress-nginx
       annotations:
         prometheus.io/port: "10254"
-        sourcegraph.prometheus/scrape: "true"
+        prometheus.io/scrape: "true"
```

Comment:
our sourcegraph prometheus instance looks for "sourcegraph.prometheus/scrape" label